test/livetest.sh

```diff
@@ -4,23 +4,11 @@
 
 # Deploy kafka client to use as producer and consumer
 # TODO: figure out if we can deploy client without topic and users first?
+kubectl create secret generic -n kafka mirror-maker-cluster-ca-cert --from-literal=ca.crt=sample-faked-cert
```

Comment:
@mtarng is this necessary for the test in it's current state?

Reply:
Yeah. The kafka client configuration expects a secret for mirror maker, and since I'm piggy-backing off the same client, we need a dummy secret.

Reply:
@mtarngCan we make t more client in the comment that we only add this because the test expects it? Also that is it a dummy cert? Otherwise this PR is fine